store/storetest/team_store.go

```diff
@@ -46,7 +46,9 @@ func TestTeamStore(t *testing.T, ss store.Store) {
 	t.Run("TeamPublicCount", func(t *testing.T) { testPublicTeamCount(t, ss) })
 	t.Run("TeamPrivateCount", func(t *testing.T) { testPrivateTeamCount(t, ss) })
 	t.Run("TeamMembers", func(t *testing.T) { testTeamMembers(t, ss) })
-	t.Run("GetMembersOrder", func(t *testing.T) { testGetMembersOrder(t, ss) })
+	t.Run("testGetMembersOrderByUserID", func(t *testing.T) { testGetMembersOrderByUserID(t, ss) })
```

Comment:
Might we use subtests within a `TestGetMembers` call instead of breaking out here? (Just for parallelism with other tests.)

Reply:
Thanks for the suggestion. I have done this now!